src/ui/components/DropdownMenu/styles.scss

```diff
@@ -81,6 +81,7 @@
   display: none;
   list-style-type: none;
   margin: 0;
+  max-width: 245px;
```

Comment:
My only thought is maybe this could be a bit wider on really large displays? Maybe for the extraLarge viewport size it could be 300?

And maybe even by default something like 260? I guess that’s a nitpick but just trying to limit when this would happen.

Or was 245 the biggest you could go to without errors?

Reply:
I got max 243 max without errors, so chose 245.
But adding 300 for `extraLarge` will be good too.